### ISSUE TITLE
Error: PostCSS plugin postcss-discard-comments requires PostCSS 8

### ISSUE THREAD

**Issue by Vinka** [SnowdogApps/magento2-frontools#448]
After upgrading to latest version 1.12 and run `npm run styles --prod` I'm getting error:
```
Starting 'styles'...
[10:26:42] 'styles' errored after 119 ms
[10:26:42] Error: PostCSS plugin postcss-discard-comments requires PostCSS 8.
Migration guide for end-users:
https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users
    at Processor.normalize (/var/www/html/richer-new/vendor/snowdog/frontools/node_modules/postcss/lib/processor.js:167:15)
    at new Processor (/var/www/html/richer-new/vendor/snowdog/frontools/node_modules/postcss/lib/processor.js:56:25)
    at postcss (/var/www/html/richer-new/vendor/snowdog/frontools/node_modules/postcss/lib/postcss.js:55:10)
    at cssnanoPlugin (/var/www/html/richer-new/vendor/snowdog/frontools/node_modules/cssnano/dist/index.js:145:31)
    at default (file:///var/www/html/richer-new/vendor/snowdog/frontools/helpers/scss.mjs:73:39)
    at file:///var/www/html/richer-new/vendor/snowdog/frontools/tasks/styles.mjs:8:17
    at Array.forEach (<anonymous>)
    at styles (file:///var/www/html/richer-new/vendor/snowdog/frontools/tasks/styles.mjs:7:12)
    at bound (domain.js:430:14)
    at runBound (domain.js:443:12)
```

Tested on many different node versions

**Comment by Igloczek** [comment]
You are using `npm` as dependency manager, right?

**Comment by Vinka** [comment]
that is correct

**Comment by Igloczek** [comment]
Sorry that it take so long, but I was on holidays.

It's now fixed and released as version 1.12.1, enjoy.

**Comment by mkwiecinska** [comment]
Thank you so much!
